Fix: linux.lua:112: attempt to call global '_l' (a nil value)

ElvisClaros opened the pull request:
The preview gives me an error which is solved with an anonymous function.
![before](https://github.com/user-attachments/assets/a59019c6-1fcb-4c7b-af91-54f234d2786e)
After the change it works perfect.
![after](https://github.com/user-attachments/assets/087bde96-9efd-429c-b04e-11e998f7c9e7)

mimikun commented:
Thank you for your contribution!

I'll try it on a Linux machine when I have time.

ElvisClaros commented:
Hi thanks to you for the plugin.
I made some more changes.
I added the possibility to preview pdf for this I use `magick` to get the first page as an image and display it with `wezterm imgcat`
I also added the possibility to preview videos with `mpv` with option `--vo=sixel --no-audio`, with `sixel` the videos are previewed great.
And finally I replaced `ls` with `exa`.
I leave a video with the changes made.


https://github.com/user-attachments/assets/fbc6bf49-756f-47b0-8e9a-d2d697fface9
